scripts/test_screeps_rl_live_dashboard.py

```
@@ -0,0 +1,210 @@
+                "officialMmoWritesAllowed": False,
+                "billingGuardBeforeScale": True,
+                "scaleDownAttempted": True,
+            },
+        },
+    )
+    (root / "rl-training").mkdir(parents=True, exist_ok=True)
+
+
+def count_rows(db_path: Path, table: str) -> int:
+    with sqlite3.connect(db_path) as conn:
+        return int(conn.execute(f"SELECT COUNT(*) FROM {table}").fetchone()[0])
```

Ruff (0.15.13)

[error] Possible SQL injection vector through string-based query construction

(S608)